README.md

```diff
@@ -1,220 +1,10 @@
-# NewWave
+# NewWaveFieldUserTask
```

Comment:
Are you sure all of the old content of the readme should be deleted?

Reply:
Ugh, no, good catch! Didn't see it. That's why I wanted to merge on develop branch first to see what's missing, and to work a bit on this pr. I kinda don't feel at home with this yet, but I see no other way for me to merge this, but to manually later fix on develop what I see missing then add to my master branch.